### PR TITLE
Update reference to windows_exporter

### PR DESCRIPTION
**Pull request by carlpett**
New repo after rename+move

### CODE REVIEW

**Comment by roidelapluie** [comment]
Thanks for your PR. For legal reasons, we require that all commits are signed with a [DCO](https://developercertificate.org/) before we can merge them. See [this blog post](https://www.cncf.io/blog/2016/11/08/cloud-native-software-can-trust/) for considerations around this.

This means that the last line of your commit message should read like:

`Signed-Off-By: Your Name <your@email.address>`

If you are using GitHub through the web interface, it's quickest to close this PR and open a new one with the appropriate line.

If you are using Git on the command line, it is probably quickest to amend and force push. You can do that with
````
git commit --amend --signoff
git push -f $remote $remote_branch_for_pr
````

As always, be careful when force-pushing.

**Comment by carlpett** [comment]
Darn Github UI... Fixed!

**Comment by roidelapluie** [comment]
Can you also fix the exporters page?

**Comment by roidelapluie** [comment]
Oh it is not listed in the exporters.. Can you add it?

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [NVIDIA GPU exporter](https://github.com/mindprince/nvidia_gpu_prometheus_exporter)
    * [ProSAFE exporter](https://github.com/dalance/prosafe_exporter)
    * [Ubiquiti UniFi exporter](https://github.com/mdlayher/unifi_exporter)
+   * [Windows/system metrics exporter](https://github.com/prometheus-community/windows_exporter)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md:76]: Just Windows will do. The system metrics for the node exporter is due to the ambiguity of "node" which has caused confusion.

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**